third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -493,6 +493,37 @@ maintenance_policy {
 
 In beta, one or the other of `recurring_window` and `daily_maintenance_window` is required if a `maintenance_policy` block is supplied.
 
+* `maintenance_exclusions_window` - (Optional) Exceptions to maintenance window. Non-emergency maintenance should not occur in these windows. A cluster can have up to three maintenance exclusions at a time [Maintenance Window and Exclusions](https://cloud.google.com/kubernetes-engine/docs/concepts/maintenance-windows-and-exclusions)
+
+Specify `start_time` and `end_time` in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) "Zulu" date format.  The start time's date is
+the initial date that the window starts, and the end time is used for calculating duration.  Specify `recurrence` in
+[RFC5545](https://tools.ietf.org/html/rfc5545#section-3.8.5.3) RRULE format, to specify when this recurs.
+Note that GKE may accept other formats, but will return values in UTC, causing a permanent diff.
+
+Examples:
```

Comment:
Do we have examples inline for other fields in container cluster? This feels out of place

Reply:
I see examples included for other attributes of 'maintenance_policy` here https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#maintenance_policy.